bindings/java/src/org/sleuthkit/datamodel/HostAddressManager.java

```diff
@@ -352,7 +515,7 @@ public void addHostNameToIpMapping(HostAddress dnsNameAddress, HostAddress ipAdd
 	 *
 	 * @throws TskCoreException
 	 */
-	List<HostAddress> getIp(String hostname) throws TskCoreException {
+	public List<HostAddress> getIp(String hostname) throws TskCoreException {
```

Comment:
Perhaps getIpAddress() to be more exact?

Reply:
renamed